src/setup_node_env/node_version_validator.test.js

```diff
@@ -68,4 +68,30 @@ describe('NodeVersionValidator', function () {
       done();
     });
   });
+
+  it('should run the script WITHOUT error when only the patch version is different', function (done) {
```

Comment:
Can we have tests for patch versions that are greater than and less than the current version?

Reply:
Fixed in [fa69e60](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1189/commits/fa69e60fe15ff10e4f9de2e2d8db7e9980de759d)